frontend/src/metabase/dashboard/components/ClickBehaviorSidebar/LinkOptions/LinkedEntityPicker/LinkedEntityPicker.tsx

```diff
@@ -175,9 +175,7 @@ export function LinkedEntityPicker({
   const handleResetLinkTargetType = useCallback(() => {
     updateSettings({
       type: clickBehavior.type,
-
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
+      // @ts-expect-error allow resetting
```

Comment:
Suppressing TS here is a hack (types are inaccurate), not an error that we expect.